metaparser/androidsignature/signature.go

```diff
@@ -128,6 +129,10 @@ func getJarSignature(path string) (string, error) {
 	// TODO: tmp debug log
 	fmt.Println(output)
```

Comment:
Remove before merge.

Reply:
Fixed in https://github.com/bitrise-io/go-android/pull/64/commits/c3059aa4427c023f98b42ded35af62488fa31103